Doc/whatsnew/3.11.rst

```diff
@@ -458,6 +458,11 @@ Deprecated
   as deprecated, its docstring is now corrected).
   (Contributed by Hugo van Kemenade in :issue:`45837`.)
 
+* The delegation of :func:`int` to :meth:`__trunc__` is now deprecated. Calling
```

Comment:
```suggestion
* The delegation of :class:`int` to :meth:`__trunc__` is now deprecated. Calling
```

Reply:
`int()` is referred as a function here, not a class.

The current text is correct.